server/services/nl.py

```diff
@@ -57,7 +259,88 @@ def _download_embeddings(self):
       _, filename = os.path.split(blob.name)
       blob.download_to_filename(os.path.join(TEMP_DIR, filename))  # Download
 
-  def search(self, query):
+  def _get_related_places(self, place_dcid):
```

Comment:
I think we should move all these data fetching and transformation to nl_interface.py, since these are highly bind to the user interface.

The Model class should just hold the minimal information for model itself? And I would keep query() to return model information as before rather than a chart_config.

Reply:
Moved all of those things to the nl_interface.py and kept only the NLP model specific things in the nl.py Model class. This where I'll add the place detection next.